### PR TITLE
Graduate admob plugin to beta

### PR DESCRIPTION
**Pull request by mit-mit**

### CODE REVIEW

File: packages/firebase_admob/README.md [hunk 1]
```diff
@@ -4,8 +4,7 @@ A plugin for [Flutter](https://flutter.io) that supports loading and
 displaying banner, interstitial (full-screen), and rewarded video ads using the
 [Firebase AdMob API](https://firebase.google.com/docs/admob/).
 
-*Warning*: This plugin is still under development, some AdMob features are not
-available yet and testing has been limited.
+*Note*: This plugin is is beta, and may still have a few issues and missing APIs.
```
Comment (mravn-google) [inline packages/firebase_admob/README.md]: Fix "is is"

**Comment by collinjackson** [comment]
Is there a reason why you choose this version number? 0.5.4+1 would be more accurate under semantic versioning since this is a documentation-only change.

**Comment by mit-mit** [comment]
I thought we used that (0.9) as a convention for packages that hit beta?

**Comment by collinjackson** [comment]
I was not aware of such a convention. If one exists I'd love to know more so I can do the same for other beta plugins. Per our conversation I'll land it as 0.5.4+1 but feel free to bump to 0.9 in the future if it makes sense to do so.

**Pull request merged**